packages/yourloops/test/integration/pages/home/home.hcp.spec.tsx

```diff
@@ -106,17 +133,35 @@ describe('HCP home page', () => {
       }
     }
 
-    const router = renderPage('/')
-    await waitFor(() => {
-      expect(router.state.location.pathname).toEqual('/home')
-    })
+    await renderHomePage()
 
     await testAppMainLayoutForHcp(appMainLayoutParams)
+  })
+
+  it('should be able to managed patients when scoped on a medical team', async () => {
```

Comment:
<!-- Nitpicks are trivial preference-based requests. These should be non-blocking by nature.  -->
**🤓 nitpick:** ‏
```suggestion
  it('should be able to manage patients when scoped on a medical team', async () => {
```